indy_client/test/cli/test_node.py

```diff
@@ -25,6 +25,32 @@ def test_add_same_node_without_any_change(be, do, newStewardCli,
     exitFromCli(do)
 
 
+def test_add_same_node_without_any_change_by_trustee(be, do, trusteeCli,
```

Comment:
Is there a test for successful editing?

Reply:
Yes, recently added tests in `indy_common/test/auth` package (test_auth_node, test_auth_nym, etc.).